release_bot/utils.py

```diff
@@ -252,3 +253,26 @@ def update_version(file, new_version):
             output.write('\n'.join(content) + '\n')
         configuration.logger.info('Version replaced.')
     return changed
+
+
+def parse_setupcfg():
+
+    """
+    Getting the data from the metadata section
+
+    :return the name from metadata or -1 for error handling
+    """
+
+    try:
+        pypi_config = configparser.ConfigParser()
+        pypi_config.read('setup.cfg')
+        metadata = pypi_config["metadata"]
+        metadata_dictionary = {} #It contains all the data from metadata section
```

Comment:
You don't need `metadata_dictionary`, `pypi_name = metadata["name"]` is just fine.

Reply:
i add the metadata_dictionary so all the section of metadata from the setup.cfg can be used in futere, should i remove it?